Sources/CGFloatExtensions.swift

```diff
@@ -62,4 +62,22 @@ extension CGFloat {
     public static func random(within: ClosedRange<CGFloat>) -> CGFloat {
         return CGFloat.random() * (within.upperBound - within.lowerBound) + within.lowerBound
     }
+    
+    /**
+      EZSE :Returns the shortest angle between two angles. The result is always between
+      -π and π.
+     
+      Inspired from : https://github.com/raywenderlich/SKTUtils/blob/master/SKTUtils/CGFloat%2BExtensions.swift
+     */
+    public static func shortestAngleInRadians(fromAngle: CGFloat, toAngle: CGFloat) -> CGFloat {
```

Comment:
Swift does not allow spaces in the names of the variables involved. The definition conflicts come because from Angle and to Angle define Angle twice. Thats why I used camel case.